tests/core/models/test_licensing.py

```diff
@@ -1568,3 +1570,205 @@ def test_repr(self, _, data_source, identifier, delivery_mechanism):
         # NOTE: we are not interested in the result returned by repr,
         # we just want to make sure that repr doesn't throw any unexpected exceptions
         repr(license_delivery_mechanism_mock)
+
+
+class TestFormatPriorities:
+
+    make = MockLicensePoolDeliveryMechanism.mechanism_of
+
+    """An arrangement of delivery mechanisms taken from a working database."""
+    sample_data_0 = [
```

Comment:
I'm not completely clear on how the above could cause one test to influence another... Could you elaborate?

Reply:
Sure. Class instances aren't recreated between test runs. For example a simple test like this: 

```python
class TestClass:

    data = ['foo', 'bar']

    def test_one(self):
        self.data.append('baz')
        assert True

    def test_two(self):
        assert self.data == ['foo', 'bar']
```

Would fail since the tests aren't isolated from each other.
```
    def test_two(self):
>       assert self.data == ['foo', 'bar']
E       AssertionError: assert ['foo', 'bar', 'baz'] == ['foo', 'bar']
E         Left contains one more item: 'baz'
E         Use -v to get the full diff
```

To be clear, the code you have isn't doing this. I just worry that if its expanded in the future, someone might unintentionally modify `sample_data_0` in one test causing side effects in other tests. Using fixtures just makes sure each of the tests are isolated. 

```python
import pytest

class TestClass:
    @pytest.fixture
    def data(self):
        return ['foo', 'bar']

    def test_one(self, data):
        data.append('baz')
        assert True

    def test_two(self, data):
        assert data == ['foo', 'bar']

```

Reply:
Ah, right, I see. Yep, will add this.